### ISSUE TITLE
[maven-metadata] Allow filtering for release versions

### ISSUE THREAD

**Issue by cromefire** [badges/shields#6995]
:clipboard: **Description**

It would be cool if the maven-metadata badge could only show the latest release version and not the latest snapshot.
This can be achieved by reading [the `release` field](https://gitlab.com/api/v4/projects/25796063/packages/maven/com/gitlab/cromefire/detekt-gitlab-report/maven-metadata.xml) or probably also by filtering for versions with no `-SNAPSHOT` suffix or even a customizable suffix (similar to the current positive matching).

**Comment by PyvesB** [comment]
Hello @cromefire ! 👋🏻 

Does https://github.com/badges/shields/pull/6594#issuecomment-857239978 provide a satisfactory way of achieving what you want?

**Comment by cromefire** [comment]
Yeah while I think it'd still be a nice feature to have this in the maven badge, the dynamic XML should work, will definitely try it.

**Comment by PyvesB** [comment]
If this becomes a regular query, anyone should feel free to resurrect #6594 and address @calebcartwright's review comments. 😉

**Comment by cromefire** [comment]
I'll just leave this open than as it kinda sounds like it's in a "PR would maybe be accepted" state if you feel like it should be closed, feel free to close it.
